### DISCUSSION TITLE
Daily 19/05

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#337] [Daily Sprint]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué haré hoy?
- Impedimentos o dificultades

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve creando las tareas que faltaban, tambien estuve creando las clases en django para la base de datos. Por la tarde asistí a la clase de programación del instituto.
Hoy estuve dandole funcionalidad a la lupa de navegación. Ahora voy a revisar si la página tiene errores y mas tarde estaré revisando la documentación y agregando el IEEE830 nuevo.

**Comment by rpgrca** [comment]
Hoy armé un submenú para el menú de administrar así todo lo del administrador queda ahí. Arreglos varios por todos lados. También tuve mi primer clase de inglés.

**Comment by BraianTroncoso** [comment]
Ayer solucioné un par de bugs, estuve en una clase BDD y también tuve clases de Ingles con el ISPC. A demas agg funcionalidades nuevas a la web como tambien mejoras en la parte de front.
